examples/multiple_metrics.cpp

```diff
@@ -81,13 +81,18 @@ int main()
         std::cout << std::endl;
     }
 
-    /// Or you can use view::to_json() for the metrics in json-format:
+    /// Or you can use view::to_json(true) for the metrics in json-format:
```

Comment:
view.to_json(true) is difficult to read. It may be better to have two to_json functions instead, one handling the true and the false functionality with appropriate names.

Reply:
IMO just always pretty print 🙂